CMakeLists.txt

```diff
@@ -489,6 +489,7 @@ check_include_file(guiddef.h HAVE_GUIDDEF_H)
 if(NOT HAVE_GUIDDEF_H)
     check_include_file(initguid.h HAVE_INITGUID_H)
 endif()
+check_include_file(dispatch/dispatch.h HAVE_DISPATCH_H)
```

Comment:
@evanmiller This should also add a case of 10.6 ppc (whether native or Rosetta), since while a header is there, `libdispatch` is unsupported for the arch.